src/main/java/info/blockchain/wallet/api/data/EthereumOptions.java

```diff
@@ -0,0 +1,26 @@
+package info.blockchain.wallet.api.data;
+
+import com.fasterxml.jackson.annotation.JsonAutoDetect;
+import com.fasterxml.jackson.annotation.JsonAutoDetect.Visibility;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import java.util.ArrayList;
+import java.util.List;
+
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@JsonIgnoreProperties(ignoreUnknown = true)
+@JsonAutoDetect(fieldVisibility = Visibility.NONE,
+    getterVisibility = Visibility.NONE,
+    setterVisibility = Visibility.NONE,
+    creatorVisibility = Visibility.NONE,
+    isGetterVisibility = Visibility.NONE)
+public class EthereumOptions {
+
+    @JsonProperty("lastTxFuse")
+    private int lastTxFuse;
```

Comment:
Bit weird that we parse as `int` and then return as `long` but I don't suppose it matters

Reply:
It is a bit weird. Doesn't matter for now, but will update it for next release